docs/root/intro/version_history.rst

```diff
@@ -7,6 +7,7 @@ Version history
 * http: mitigated a race condition with the :ref:`delayed_close_timeout<envoy_api_field_config.filter.network.http_connection_manager.v2.HttpConnectionManager.delayed_close_timeout>` where it could trigger while actively flushing a pending write buffer for a downstream connection.
 * redis: add support for zpopmax and zpopmin commands.
 * upstream: added :ref:`upstream_cx_pool_overflow <config_cluster_manager_cluster_stats>` for the connection pool circuit breaker.
+* ext_authz: added a `x-envoy-auth-partial-body` metadata header set to `0|1` indicating if there is a partial body sent in the authorization request message.
```

Comment:
Can you use alphabetical order here, please?

Reply:
fixed

Reply:
ok, and also merged with master to fix the conflict

Reply:
lol, and that merge destroyed the alpha order I just fixed, so fixed it again. Sigh.

@gsagula Looks like that invalidated your review, too. Sorry.